pkg/plugin/v3/webhook.go

```diff
@@ -114,7 +114,8 @@ func (p *createWebhookPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
 
 	// Create the actual resource from the resource options
 	res := p.resource.NewResource(p.config, false)
-	return scaffolds.NewWebhookScaffolder(p.config, string(bp), res, p.defaulting, p.validation, p.conversion), nil
+	return scaffolds.NewWebhookScaffolder(p.config, string(bp), res, p.defaulting, p.validation, p.conversion),
+		nil
```

Comment:
What does this change have to do with this PR?

Reply:
Accidental enter, corrected it